src/Psalm/Internal/Analyzer/Statements/Expression/Call/ArgumentAnalyzer.php

```diff
@@ -1505,6 +1505,14 @@ private static function processTaintedness(
             return $input_type;
         }
 
+        // numeric types can't be tainted
+        if ($statements_analyzer->data_flow_graph instanceof TaintFlowGraph
+            && $input_type->isSingle()
+            && ($input_type->isInt() || $input_type->isFloat())
```

Comment:
Same for bool?

Reply:
Probably, and there's surely more! We'll gladly accepts PRs :)